openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -61,6 +61,8 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             -xz_compression $(BR2_OPENPOWER_PNOR_XZ_ENABLED)
 
         mkdir -p $(STAGING_DIR)/pnor/
+
+    if $(BR2_BOOTKERNEL_ENABLED); then \
```

Comment:
Rather than duplicating the whole blob, you should be able to do this by:

$(if $(findstring "true",$(BR2_BOOTKERNEL_ENABLED)),-bootkernel $(BINARIES_DIR)/$(BR2_LINUX_KERNEL))